src/main/java/seedu/address/logic/commands/SortCommand.java

```diff
@@ -15,7 +15,7 @@
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Sorts all clients based on parameters given.\n"
             + "Parameters available: meeting, name, prev, salary \n"
-            + "Example: " + COMMAND_WORD + " John Doe";
+            + "Example: " + COMMAND_WORD + " name";
```

Comment:
This is a usage example, it would be better to type the actual command instead. E.g. `Example: sort name`

Reply:
Sorry could I clarify what you mean by this? The current help shows `Example: sort name`. Do you mean change `COMMAND_WORD` to sort?

Reply:
`COMMAND_WORD` is not a string, in this case it will output `Example: sort name`. Will be resolving this and approving the PR.